modules/azure/src/main/java/org/testcontainers/azure/AzureEventhubsEmulatorContainer.java

```diff
@@ -0,0 +1,106 @@
+package org.testcontainers.azure;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.MountableFile;
+
+/**
+ * Testcontainers implementation for Azure Eventhubs Emulator.
+ * <p>
+ * Supported image: {@code "mcr.microsoft.com/azure-messaging/eventhubs-emulator"}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>AMQP: 5672</li>
+ *     <li>Kafka: 9092</li>
+ * </ul>
+ */
+public class AzureEventhubsEmulatorContainer extends GenericContainer<AzureEventhubsEmulatorContainer> {
+
+    private static final int DEFAULT_AMQP_PORT = 5672;
+
+    private static final int DEFAULT_KAFKA_PORT = 9092;
```

Comment:
kafka port is not used. Can we add a test? Otherwise, let's remove it.

Reply:
Thanks! Sure, I will add a test tomorrow.